src/autocomplete/styles.css.ts

```diff
@@ -5,6 +5,7 @@ export default css`
 		display: block;
 		position: relative;
 		min-width: 35px;
+		font-family: 'Inter', sans-serif;
```

Comment:
If you do this, you are breaking the cascade, forcing authors to either load the Inter font OR write `cosmoz-autocomplete { font-family: myfont-family, etc }`. Revert this.

Reply:
Addressed